pyproject.toml

```diff
@@ -26,6 +26,8 @@ urllib3="1.25.9"
 Werkzeug="1.0.1"
 
 [tool.poetry.dev-dependencies]
+pytest="5.4.3"
+selenium="3.141.0"
```

Comment:
It kind of looks like you added this manually since there are no spaces...

Did you know you could have run e.g. `poetry add pytest selenium --dev` to install them and automatically update this file?